src/f32-vbinary/gen/f32-vmax-hvx-u64.c

```diff
@@ -0,0 +1,65 @@
+// Auto-generated file. Do not edit!
+//   Template: src/f32-vbinary/vop-hvx.c.in
+//   Generator: tools/xngen
+//
+
+#include <assert.h>
+
+#include <hvx_hexagon_protos.h>
+#include <hexagon_protos.h>
+#include <hexagon_types.h>
+
+#include <xnnpack/common.h>
+#include <xnnpack/math.h>
+#include <xnnpack/vbinary.h>
+
+void xnn_f32_vmax_ukernel__hvx_u64(
+    size_t batch,
+    const float* input_a,
+    const float* input_b,
+    float* output,
+    const union xnn_f32_default_params params[restrict XNN_MIN_ELEMENTS(1)]) XNN_OOB_READS
```

Comment:
Could you mask the remainder read so you can remove OOB_READs?  (Out of Bounds)
HVX has a large over read, and the masked remainder wont normally be a performance bottleneck... the main loop will be.  On AVX512, RiSCV and SVE I expect we'll move towards asan friendly reads using predicates.